src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java

```diff
@@ -285,6 +301,27 @@ private void parseSiegeWarNationCommand(Player player, String[] args) {
 		}
 	}
 
+	private void parseSiegeWarTownCommand(Player player, String[] args) {
+		if (!player.hasPermission(SiegeWarPermissionNodes.SIEGEWAR_COMMAND_SIEGEWAR_TOWN.getNode(args[0]))) {

```

Comment:
This permission test might require that the node also be added as a child node to the siegewar.command.town.* node in the plugin.yml.